### PR TITLE
feat(server): add an option to enable/disable dynamic model loading

### PR DESCRIPTION
**Pull request by kkaneda**

### CODE REVIEW

File: deployments/server/values.yaml [hunk 1]
```diff
@@ -148,6 +148,10 @@ certificate:
     # The default value is Issuer (i.e. a locally namespaced Issuer)
     kind: ClusterIssuer
 
+requestRouting:
+  # Specify whether dynamic on-demand model loading is enabled.
```
Comment (Ladicle) [inline deployments/server/values.yaml:152]:
```suggestion
  # Specify whether dynamic on-demand model loading is enabled.
  # If set to false, the request will fail immediately if the specified
  # model has not yet been loaded on any cluster.
```

**Review by Ladicle — APPROVED** [review]
LG

It seems the values schema is not generated.

File: deployments/server/values.yaml (hunk 1 above)
Comment (kkaneda) [reply deployments/server/values.yaml:152]: done.

**Comment by kkaneda** [comment]
@Ladicle , how did you generate the values schema? I followed https://www.arthurkoziel.com/validate-helm-chart-values-with-json-schemas/

**Comment by Ladicle** [comment]
make generate-chart-schema (or make lint)

**Comment by kkaneda** [comment]
thx!

**Pull request merged**